lib/Command/Clean.php

```diff
@@ -50,17 +50,17 @@ class Clean extends Command {
 	protected RowService $rowService;
 	protected TableService $tableService;
 	protected LoggerInterface $logger;
-	protected RowMapper $rowMapper;
+	protected LegacyRowMapper $rowMapper;
 
 	private bool $dry = false;
 	private int $truncateLength = 20;
 
-	private ?Row $row = null;
+	private ?LegacyRow $row = null;
 	private int $offset = -1;
 
 	private OutputInterface $output;
 
-	public function __construct(LoggerInterface $logger, ColumnService $columnService, RowService $rowService, TableService $tableService, RowMapper $rowMapper) {
+	public function __construct(LoggerInterface $logger, ColumnService $columnService, RowService $rowService, TableService $tableService, LegacyRowMapper $rowMapper) {
```

Comment:
This command works on (or with) information from the legacy row mapper, which does not know about new data, right? If that is still necessary.